task_templates/README.md

```diff
@@ -1,50 +1,73 @@
 # Templates for Custom Tasks
+
 In this directory are three folders - estimators, transforms, and pipelines
 
 * estimators - meant to be used as the prediction step in a blueprint
-* transforms - meant to be used as a preprocessing step
-* pipelines - meant to be added as a single task blueprint, these have some preprocessing and an estimator built in
+* transforms - meant to be used as a preprocessing step in a blueprint
+* pipelines - meant to be added as a single task blueprint, these have some preprocessing and an
+  estimator built in
 
-Our estimator examples are also different than the pipelines examples because, we decided to implement the score
-hooks for all the estimator templates, and for the pipelines templates we did not. You only don't have to use a score
-hook if your model inherits from sklearn, pytorch, keras, or xgboost and doesn't need any additional functionality
+Our estimator examples are also different than the pipelines examples because these templates have
+score hooks implemented. The pipelines examples do not have score hooks to demonstrate the automatic
+scoring functionality of the DRUM middleware layer.
```

Comment:
I think you should readd the sklearn/pytorch/etc piece here because most users probably won't go through all the DRUM docs to konw what the middleware does